udemy_exercises/cell_phone/Contact.java

```diff
@@ -1,7 +1,50 @@
-package udemy_exercises.cell_phone;
+package cell_phone;
 
 class Contact {
+	
+private String name;
+private String phoneNumber;
 
+    public Contact(String name, String phoneNumber) {
+        this.name = name;
+        this.phoneNumber = phoneNumber;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public void setPhoneNumber(String phoneNumber) {
+        this.phoneNumber = phoneNumber;
+    }
+    
+    public Contact saveContact(String name, String phoneNumber) {
```

Comment:
Should this be the responsibility of Contact?  Seems to conflict a bit with the approach of keeping a Contact as a container, allowing for the updating of both the name and the number.  This method is going to return a whole new copy of this exact Contact, which could introduce duplicate records into the system, breaking some of the assumptions made in other classes.